cvat/apps/engine/cache.py

```diff
@@ -58,11 +64,19 @@ def get_local_preview_with_mime(self, frame_number, db_data):
 
         return item
 
-    def get_cloud_preview_with_mime(self, db_storage):
-        item = self._get_or_set_cache_item(
-            key=f'cloudstorage_{db_storage.id}_preview',
-            create_function=lambda: self._prepare_cloud_preview(db_storage)
-        )
+    def get_cloud_preview_with_mime(
+        self,
+        db_storage,
+        just_check: bool = False
```

Comment:
From my point of view, separate `get` method is better

Reply:
Updated